File: problematica.py
# se solicita que se eliminen los nuemros duplicados
# solo se deven mostrar los numeros unicos
def eliminar_duplicados(A: list[int]) -> list[int]:
    unicos = set(A)
    return [num for num in unicos]

print(eliminar_duplicados([2,3,4,3,2,2,1]))

File: problematica.kt
// se solicita que se eliminen los nuemros duplicados
// solo se deven mostrar los numeros unicos
fun eliminarDuplicados(lista: List<Int>): List<Int> {
    val unicos = lista.toSet()

    return unicos.toList()
}

fun main() {
    val numerosConDuplicados = listOf(2, 3, 4, 3, 2, 2, 1)
    val numerosUnicos = eliminarDuplicados(numerosConDuplicados)
    println(numerosUnicos) 
}